# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/IPCIdentityCredential.ipdlh
# Commit: da37e2d8a175
# Full Hash: da37e2d8a175fda08a98920d820dda0899b6aedf
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 15b92f98f62c -r da37e2d8a175 dom/credentialmanagement/identity/IPCIdentityCredential.ipdlh
--- a/dom/credentialmanagement/identity/IPCIdentityCredential.ipdlh	Thu Jun 12 18:16:52 2025 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,24 +0,0 @@
-/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-include "mozilla/dom/PermissionMessageUtils.h";
-[RefCounted] using class nsIPrincipal from "nsIPrincipal.h";
-
-namespace mozilla {
-namespace dom {
-
-// Most of these fields are taken directly from mozilla::dom::IdentityCredentialInit
-// Others are:
-//    identityProvider: the principal that created the IdentityCredential this represents.
-//    infoExpiresAt: PR_Now() + the mozilla::dom::IdentityCredentialInit's infoExpiresAfter, in ms
-struct IPCIdentityCredential
-{
-  nsString id;
-  nsString? token;
-};
-
-}
-}